draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2394,21 +2409,30 @@ for random oracle encodings.
 
 {{hashtobase}} describes considerations for uniformly hashing to field elements.
 
+When the hash\_to\_curve function ({{roadmap}}) is instantiated
+with hash\_to\_base ({{hashtobase}}), the resulting function is
+indifferentiable from a random oracle.
+In most cases such a function can be safely used in protocols whose security
+analysis assumes a random oracle that outputs points on an elliptic curve.
+As Ristenpart et al. discuss in {{RSS11}}, however, the indifferentiability
```

Comment:
Since we don't define indifferentiability framework here, might it make sense to say something such as, "As Ristenpart et al. discuss in {{RSS11}}, however, security proofs in the Random Oracle model may not always hold when such functions are replaced with indifferentiable counterpart," or something? (That is, perhaps unroll the meaning of the framework inline so readers don't have to look at RSS11.)

Reply:
Good call! Better now?

Reply:
Yep — perfect!